### DISCUSSION TITLE
Get wallet address by social login.

### DISCUSSION THREAD

**Discussion by ppoz** [Web3Auth/Web3Auth#627] [Help]
I am making a game with web3auth social login and I want a wallet for users to get or transfer nft from game. so how can I get the wallet address for social login for every user is it possible or not?

**Comment by rinzler99** [comment]
You can use web3auth sdk to create a wallet by social logins and torus wallet ui plugin to transfer nfts and tokens. Please find below links for the reference:
Demo: https://demo-app.web3auth.io/
Doc web3auth sdk: https://web3auth.io/docs/sdk/web/
Torus wallet ui plugin: https://web3auth.io/docs/sdk/web/plugins/torus-wallet

**Reply by ppoz** [reply]
do you know how to install the torus wallet Ui plugin for unity and c#?

**Reply by rinzler99** [reply]
Hey plugins are only supprted in web right now.

**Reply by ppoz** [reply]
Ok, thanks anyway. Is there any way to transfer nft from the open-sea marketplace, to Social login from web3auth, or to web3auth SDK for unity c#?

**Comment by himanshuchawla009** [comment]
When user login with web3auth it returns user's dapp scoped private key, you can use this key to generate user's wallet with the corresponding blockchain library.

**Reply by ppoz** [reply]
Thanks Bro

**Comment by Oberhaim** [comment]
It feels very shady to manipulate private keys to get the public address... any work around ?

**Reply by marcus-sa** [reply]
I agree
